libr/core/anal.c

```diff
@@ -541,29 +541,34 @@ static int cb(void *p, const char *k, const char *v) {
 	case '*':
 		HINTCMD_ADDR (hint, arch, "aha %s");
 		HINTCMD_ADDR (hint, bits, "ahb %d");
-		HINTCMD_ADDR (hint, size, "ahl %d");
+		HINTCMD_ADDR (hint, size, "ahs %d");
 		HINTCMD_ADDR (hint, opcode, "aho %s");
-		HINTCMD_ADDR (hint, opcode, "ahs %s");
```

Comment:
Was there any reason to have "opcode" duplicated in this and in the next line, but with different ```ah``` commands?